packages/data-transforms/src/track-value-array-to-segments/track-value-array-to-segments.test.ts

```diff
@@ -0,0 +1,34 @@
+import { trackValueArrayToSegments } from './track-value-array-to-segments'
+import { Field } from './types'
+
+describe('trackValueArrayToSegments', () => {
+  test('parses elevation properly', () => {
+    const valueArray = [
+      -2147483648, 1, 0, -42342480, 47021440, 1562724078, 10300000, -3943, 270600000, 1, 782340,
```

Comment:
why you are not using DEFAULT_NULL_VALUE here?

Reply:
Because here I'm mocking the response from the api endpoint in valueArray format. We shouldn't use it here since we don't know what we're going to receive (in theory).